aws/chamber/user.tf

```diff
@@ -1,13 +1,16 @@
 # Chamber user for CI/CD systems that cannot leverage IAM instance profiles
 # https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-paramstore-access.html
 module "chamber_user" {
-  source        = "git::https://github.com/cloudposse/terraform-aws-iam-chamber-user.git?ref=tags/0.1.4"
-  namespace     = "${var.namespace}"
-  stage         = "${var.stage}"
-  name          = "chamber"
-  attributes    = ["codefresh"]
-  kms_key_arn   = "${module.chamber_kms_key.key_arn}"
-  ssm_resources = ["${format("arn:aws:ssm:%s:%s:parameter/kops/*", var.region, var.account_id)}"]
+  source      = "git::https://github.com/cloudposse/terraform-aws-iam-chamber-user.git?ref=tags/0.1.4"
```

Comment:
Update to the latest version:
https://github.com/cloudposse/terraform-aws-iam-chamber-user.git?ref=tags/0.1.5